src/lib/components/Snackbar/_styles.scss

```diff
@@ -0,0 +1,120 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/settings/global';
+@import '../../../styles/tools/breakpoints';
+
+web-snackbar {
+  align-items: center;
+  bottom: 0;
+  display: block;
+  justify-content: center;
+  left: 0;
+  margin: 0;
+  pointer-events: none;
+  position: fixed;
+  right: 0;
+  visibility: hidden;
+  z-index: 300;
+  
+  @include bp(sm) {
+    margin: 8px;
+  }
+
+  .web-snackbar__surface {
+    align-items: center;
+    background-color: $GREY_900;
+    display: flex;
+    justify-content: flex-start;
+    max-width: 400px;
+    min-width: 344px;
+    opacity: 0;
+    transform: scale(.8);
+    
+    @include bp(sm) {
+      border-radius: $GLOBAL_RADIUS;
+      box-shadow: 0 3px 5px -1px rgba(0, 0, 0, .2),
+                  0 6px 10px 0 rgba(0, 0, 0, .14),
+                  0 1px 18px 0 rgba(0, 0, 0, .12);
+    }
+  }
+
+  .web-snackbar__label {
+    color: rgba($WHITE, .87);
+    font-size: .875rem;
+    padding: 14px 16px;
+  }
+
+  .web-snackbar__actions {
+    margin-left: 0;
+    margin-right: 8px;
+    display: flex;
+    align-items: center;
+  }
+
+  .web-snackbar__action {
+    font-size: .875rem;
+    color: $WEB_SECONDARY_COLOR;
+    height: auto;
+  }
+
+  .web-snackbar__action:hover,
+  .web-snackbar__action:focus,
+  .web-snackbar__action:active {
+    box-shadow: none;
+  }
+
+  .web-snackbar__action:hover {
+    background: rgba($WEB_SECONDARY_COLOR, .08);
+  }
+
+  .web-snackbar__action:focus {
+    background: rgba($WEB_SECONDARY_COLOR, .12);
+  }
+
+  .web-snackbar__action:active {
+    background: rgba($WEB_SECONDARY_COLOR, .16);
+  }
+
+  .web-snackbar__action + .web-snackbar__action {
+    margin-left: 8px;
+  }
+}
+
+web-snackbar[open] {
+  visibility: visible;
+
+  .web-snackbar__surface {
+    transform: scale(1);
+    opacity: 1;
+    pointer-events: auto;
+  }
+}
+
+// Opening animation
+web-snackbar[open][animatable] {
+  .web-snackbar__surface {
+    transition: opacity 150ms cubic-bezier(0, 0, .2, 1),
+                transform 150ms cubic-bezier(0, 0, .2, 1);
+  }
+}
+
+// Closing animation
+web-snackbar[animatable] {
+  visibility: visible;
+
+  .web-snackbar__surface {
+    transition: opacity 75ms cubic-bezier(.4, 0, 1, 1),
+                transform 75ms cubic-bezier(0, 0, .2, 1);
+  }
+}
+
+web-snackbar[stacked] {
```

Comment:
Why is this an attribute? Do you plan to have different snackbars?

Reply:
I was copying the Material Components version. If you remove it, it should all layout horizontally. I was thinking we might use that for the "We have a new service worker, hit refresh!" notifications, if we choose to do that pattern.

![image](https://user-images.githubusercontent.com/1066253/65557576-c0352c80-dee8-11e9-8913-9aed1f19b9f0.png)